frontend/src/main/scala/bloop/data/Project.scala

```diff
@@ -303,8 +301,10 @@ object Project {
     val metricsDirOption = ("-YhydraMetricsDirectory", hydraMetricsDir.syntax)
     val hydraTagOption = ("-YhydraTag", hydraTag)
     val sourcepathOption = ("-sourcepath", hydraSourcepath)
-    val sourcePartitionerOption = ("-YsourcePartitioner", hydraSourcePartitioner)
+    val sourcePartitionerOption = (s"-YsourcePartitioner:$hydraSourcePartitioner", "")
 
+    // FIXME: Make the number of workers configurable (2 should stay the default)
+    val hydraCpus = ("-cpus", "2")
```

Comment:
That works for me!

Reply:
Alright, I've included a system property as it was the simpler and quickest thing I could do. However, I'd still like to include a CLI option as the one you suggested, but this can come in a separate PR. I checked how `--pipeline` is processed in `Bloop.scala` and `BloopBspServices.scala`, and I guess I need to do something similar to support a configurable number of Hydra cpus. With the additional complexity in `Bloop.scala` that I should handle the combination of `--pipeline` with `--hydra-cpus` as the current implementation of the CLI options parsing is fairly ad-hoc. Does it sound right?